_author_instructions/01_policies.md

```diff
@@ -180,25 +180,27 @@ This is handled through Stripe Connect and can be paid via any major credit card
 
 ## The review process
 
-The review process generally begins with an editor reviewing your document to check that it is ready for review (e.g., that it uses suitable English, is laid out appropriately and legibly, etc.) and subsequently selecting reviewers.
-If there are major issues at this stage, your article may be returned for revision prior to review.
-Reviews will generally be anonymous, though reviewers will be allowed to make themselves known if they desire.
```

Comment:
Why are we omitting this?  I think it's useful info for authors

Reply:
The idea was to make this information more "reviewer instructions" than a description to the author, since we needed reviewer instructions, and I wanted to reduce duplication. But maybe this make it less readable.

Reply:
It IS called editorial policies, right?

Reply:
So maybe we need a separate page on reviewer instructions?

Reply:
Well, the page does say "This article describes how LiveCoMS article preparation and submission work, and provides details covering the review process, revisions, authorship, and other aspects."  So I guess it's more than policies.

Reply:
I was just making the point that more goes on this page than just reviewer instructions, so you probably don't want to remove this content unless you are migrating it somewhere else.